src/Uno.UI.Toolkit/ElevatedView.cs

```diff
@@ -0,0 +1,143 @@
+﻿using System;
+using Windows.UI;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Markup;
+using Windows.UI.Xaml.Media;
+#if __IOS__
+using CoreGraphics;
+using _View = UIKit.UIView;
+#elif __MACOS__
+using CoreGraphics;
+using _View = AppKit.NSView;
+#endif
+
+
+namespace Uno.UI.Toolkit
+{
+	[ContentProperty(Name = "Content")]
+	[TemplatePart(Name = "PART_Border", Type = typeof(Border))]
+	public sealed partial class ElevatedView : Control
+#if !NETFX_CORE
+		, ICustomClippingElement
+#endif
+	{
+		/*
+		 *  +-ElevatedView------------+
+		 *  |                         |
+		 *  |  +-Border------------+  |
+		 *  |  |                   |  |
+		 *  |  |  +-Content-----+  |  |
+		 *  |  |  | (...)       |  |  |
+		 *  |  |  +-------------+  |  |
+		 *  |  |                   |  |
+		 *  |  +-------------------+  |
+		 *  |                         |
+		 *  +-------------------------+
+		 *
+		 * Elevated is responsible for the shadow
+		 * Border responsible for rounded corners (if any)
+		 *
+		 */
+
+		private Border _border = new Border();
+
+		public ElevatedView()
+		{
+			DefaultStyleKey = typeof(ElevatedView);
+		}
+
+		protected override void OnApplyTemplate()
+		{
+			_border = GetTemplateChild("PART_Border") as Border;
+
+			UpdateElevation();
+		}
+
+		public static readonly DependencyProperty ElevationProperty = DependencyProperty.Register(
+			"Elevation", typeof(double), typeof(ElevatedView), new PropertyMetadata(default(double), OnChanged));
+
+#if __ANDROID__
+		public new double Elevation
+#else
+		public double Elevation
+#endif
+		{
+			get => (double)GetValue(ElevationProperty);
+			set => SetValue(ElevationProperty, value);
+		}
+
+		public static readonly DependencyProperty ShadowColorProperty = DependencyProperty.Register(
+			"ShadowColor", typeof(Color), typeof(ElevatedView), new PropertyMetadata(Color.FromArgb(64, 0, 0, 0), OnChanged));
+
+		public Color ShadowColor
+		{
+			get => (Color)GetValue(ShadowColorProperty);
+			set => SetValue(ShadowColorProperty, value);
+		}
+
+		public static readonly DependencyProperty ContentProperty = DependencyProperty.Register(
+			"Content", typeof(object), typeof(ElevatedView), new PropertyMetadata(default(object)));
+
+		public object Content
+		{
+			get => GetValue(ContentProperty);
+			set => SetValue(ContentProperty, value);
+		}
+
+#if !NETFX_CORE
+		public new static readonly DependencyProperty BackgroundProperty = DependencyProperty.Register(
+			"Background", typeof(Brush), typeof(ElevatedView), new PropertyMetadata(default(Brush)));
```

Comment:
Should'nt it be possible to set the background/cornerradius properties in UWP (even if it's not supported) ? It'll make it difficult to conditionally exclude the behavior.

Reply:
Yes it's possible...  it's just the property exists on UWP :-)